src/main/java/com/epam/spring/time_tracking/repository/UserActivityRepo.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.spring.time_tracking.repository;
+
+import com.epam.spring.time_tracking.model.UserActivity;
+
+import java.util.List;
+
+public interface UserActivityRepo {
+
+    List<UserActivity> getActivityUsers(int activityId);
+
+    List<UserActivity> getActivitiesForUser(int userId);
```

Comment:
No need to specify for whom this activity, we understand it from class name, just use getActivitiesByUserId() instead